src/main/java/core/basesyntax/file/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.file;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String fileName) {
+        try {
+            Files.write(Paths.get(fileName), data.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file" + fileName, e);
```

Comment:
According to the checklist, you should not ignore exceptions in try-catch blocks, and you are correctly rethrowing a RuntimeException. However, the exception message should be more informative. Include a space before the file name in the exception message to enhance readability.